clients/python-wrapper/pyproject.toml

```
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lakefs"
+version = "0.14.0"
+description = "lakeFS Python SDK Wrapper"
+keywords = ["OpenAPI", "OpenAPI-Generator", "lakeFS API", "Python Wrapper"]
+requires-python = ">=3.9"
+readme = { file = "README.md", content-type = "text/markdown" }
+license = { text = "Apache-2.0" }
```

Consider using [PEP 639](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license-and-license-files) based `license` expression (which is supported with `setuptools>=77`).


```suggestion
license = "Apache-2.0"
```